App.tsx

```diff
@@ -1,109 +1,61 @@
-import React from 'react';
-import { Image, SafeAreaView, StyleSheet, Text, View } from 'react-native';
-import { SimpleLineIcons } from '@expo/vector-icons';
+import * as React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { LoginScreen } from './src/features/login';
+import { ProfileScreen } from './src/features/profile';
+import { SettingsScreen } from './src/features/settings';
+import { SwiperScreen } from './src/features/swiper';
 
-import { Button } from './src/components/button';
-import { InlineButton } from './src/components/inline-button';
-import { Input } from './src/components/input';
+import { TabBarIcon } from './src/components/tab-bar-icon';
 
-import loginBackground from './assets/login-background.jpg';
+const Tab = createBottomTabNavigator();
 
 export default function App() {
   return (
-    <SafeAreaView style={styles.container}>
-      <Image
-        source={loginBackground}
-        resizeMode="cover"
-        style={styles.background}
-      />
-      <Text style={styles.title}>Welcome</Text>
-      <View style={styles.login}>
-        <View style={styles.header}>
-          <Text style={styles.headerTitle}>Login</Text>
-          <InlineButton title="SIGN UP" />
-        </View>
-        <Input icon="user" placeholder="Login" />
-        <Input icon="lock" placeholder="Password" secure />
-        <InlineButton title="Forget Password?" />
-        <Button title="Login" customStyle={styles.button} />
-      </View>
-      <View style={styles.footer}>
-        <Text>Login with:</Text>
-        <SimpleLineIcons
-          style={styles.social}
-          name="social-facebook"
-          size={24}
-          color="black"
+    <NavigationContainer>
+      <Tab.Navigator
+        screenOptions={({ route }) => ({
+          tabBarShowLabel: false,
+          headerShown: false,
+        })}
+      >
+        <Tab.Screen
+          name="Login"
+          component={LoginScreen}
+          options={{
+            tabBarIcon: ({ focused }) => (
+              <TabBarIcon name="login" focused={focused} />
+            ),
+          }}
```

Comment:
Login shouldn't be one of the tab screens.

Reply:
done